lib/tasks/specs.rake

```diff
@@ -1,5 +1,31 @@
 namespace :ofn do
   namespace :specs do
+    namespace :run do
+      def spec_folders
+        Pathname("spec/").children.select(&:directory?).map { |p|
+          p.split.last.to_s
+        }.reject { |p| %w(support factories javascripts performance).include?(p) }
+      end
+
+      def execute_rspec_for_pattern(pattern)
+        system "bundle exec rspec --profile --pattern \"#{pattern}\""
+      end
+
+      def execute_rspec_for_spec_folder(folder)
+        execute_rspec_for_pattern("spec/#{folder}/{,/*/**}/*_spec.rb")
+      end
+
+      def execute_rspec_for_spec_folders(folders)
+        folders = folders.join(",") if folders.is_a?(Array)
```

Comment:
I believe `folders = folders.join(",")` alone would have the same result.

Reply:
I leave this here because I think it's worth changing this :point_up: Then we can merge.

Reply:
I was guarding against the case where the argument was a string, like "foo,bar,baz", but given that this is such a limited use case I think it's fine to assume it's an array.